### ISSUE TITLE
Debugging CppCustomVisualizer doesn't work in VS2019

### ISSUE THREAD

**Issue by pps83** [microsoft/ConcordExtensibilitySamples#54]
If I try to debug (F5) CppCustomVisualizer project new VS2019 instance starts but doesn't load TargetApp.sln. Even if I load it manually from started VS2019 instance, FILETIME in debugger doesn't doesn't get displayed as intended by CppCustomVisualizer.
This is how debugging page of Debug configuration of vsix project looks like:
![image](https://user-images.githubusercontent.com/1614246/72277858-eed44000-365c-11ea-9df1-c5f24d301959.png)

Note, that Release config is identical to the [screenshot in Greg's comment](https://github.com/microsoft/ConcordExtensibilitySamples/issues/52#issuecomment-572173848_).
I use pretty much barebones latest VS2019 with little no none extensions installed. If I build CppCustomVisualizer and manually install the resulting extension, then yes, FILETIME visualizer works in TargetApp. I do not have any local changes in the ConcordExtensibilitySamples clone and the path doesn't have any spaces.

_Originally requested by @gregg-miskelly in https://github.com/microsoft/ConcordExtensibilitySamples/issues/52#issuecomment-572173848_

**Comment by gregg-miskelly** [comment]
Thanks for opening this. I enhanced the documentation by adding this section: https://github.com/microsoft/ConcordExtensibilitySamples/wiki/Cpp-Custom-Visualizer-Sample#if-debug-launch-doesnt-start-visual-studio-correctly

**Comment by pps83** [comment]
When I open clean checkout debug of vsix looks like on your screenshot. After running it once, it changes to what I have. Regardless, if I start it for the first time, or if I force `/rootsuffix Exp $(SolutionDir)\TargetApp\TargetApp.sln` (or `/rootsuffix Exp $(SolutionDir)TargetApp\TargetApp.sln`) I get identical behavior: new instance of VS starts and shows dialog to load project/solution and even if I load TargetApp manually in that new VS instance I don't get visualizer working.

**Comment by pps83** [comment]
I figured it out why it didn't work for me. I use awesome [Smart Command Line Arguments](https://github.com/MBulli/SmartCommandlineArgs) extension by @MBulli and this extension clears/resets default `Command Arguments` string with an empty string. That's clearly a bug in SmartCommandlineArgs. When I set `/rootsuffix Exp $(SolutionDir)\TargetApp\TargetApp.sln` in command args windows (part of `Smart Command Line Arguments` extension) for vsix project, then everything work. Thanks for your help and sorry for taking your time!
